lib/matplotlib/ft2font.pyi

```diff
@@ -232,23 +215,73 @@ class FT2Font:
     def set_text(
         self, string: str, angle: float = ..., flags: int = ...
     ) -> NDArray[np.float64]: ...
```

Comment:
I'm wondering if `np.double` is the correct return type?

https://numpy.org/doc/stable/reference/arrays.scalars.html#numpy.double

From reading the documentation it seems like float64 is an alias for double on *some* (probably most though) platforms.

(Not added in the PR, but the missing reference is...)

Reply:
Good thought, but unfortunately it doesn't fix the missing reference. It appears that Sphinx doesn't read any `.pyi` files (which is why most other pages don't have type hint issues), but in this case is seeing something that `pybind11` is generating in the runtime itself.